lib/galaxy/tools/__init__.py

```diff
@@ -2352,8 +2347,8 @@ def to_json (self, trans, kwd={}, is_workflow=False):
         tool_message = ''
         if job:
             try:
-                job_params = job.get_param_values( trans.app, ignore_errors = True )
-                job_messages = self.check_and_update_param_values( job_params, trans, update_values=False )
+                job_params = job.get_param_values( trans.app, ignore_errors=True )
+                self.check_and_update_param_values( job_params, trans, update_values=False )
```

Comment:
Thanks @guerler for the details!

@jmchilton @dannon: ready to merge or is there anything else that needs to be fixed?